### ISSUE TITLE
Advanced Search Selectors - 3.3.2 Labels or Instructions - 1 select missing label present on the 'like/notlike/equals' select2 dropdown

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#10106]
As a user reliant on the screen reader to understand the name and purpose of a dropdown, I expect each dropdown to have a label so that I can deduce from a title what kind of options I can expect in the dropdown.

**Linked Work Item:**
#9386 

**Expected Result:**
A select component should have a label so that users reliant on a screen reader can be told the name and purpose of the component.

**Actual Result:**
This select2 dropdown did not have a <label> attribute associated with it in the HTML and therefore has a non-existent screen reader interpretation.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/72240bb3-4ca3-452c-902d-6a017fdb74a3)

**Comment by johnatawnclementawn** [comment]
@imogenroberts this should be resolved in the latest

**Comment by apeters** [comment]
fixed in the latest

**Comment by imogenroberts** [comment]
RETEST: No select components are missing labels in the Advanced Search Panel. 

**Evidence:**
![Advanced Search - 3 3 2 Labels and Instructions FIXED ](https://github.com/archesproject/arches/assets/77161085/71cc5d4e-d2b5-4b7a-8fa9-ba70d14f0265)
